### PR TITLE
update (docs) Diana Pattern Options and Instructions

### PR DESCRIPTION
**Pull request by bobgeorgethe3rd**
Was removing some old diagrams from the pattern options and noticed that the instructions needed a re-format

### CODE REVIEW

File: markdown/org/docs/patterns/diana/instructions/en.md [hunk 1]
```diff
@@ -1,63 +1,67 @@
...
 
-## Close shoulder seams
+### Step 2: Close shoulder seams
 
 ![This drawing was too nice not to use](neckline.jpg)
 
-Finish the raw edge on the front neckline in a way that suits your fabric 
-(if it doesn’t fray, you can leave it unfinished). 
+- Finish the raw edge on the front neckline in a way that suits your fabric (if it doesn’t fray, you can leave it unfinished).
+- With *good sides together*, place the front on the back along the shoulder seams.
```
Comment (nicholasdower) [inline markdown/org/docs/patterns/diana/instructions/en.md:23]: Small suggestion: I don't find "along the shoulder seams" clearer than "aligning the shoulder seams".

File: markdown/org/docs/patterns/diana/instructions/en.md [hunk 2]
```diff
@@ -1,63 +1,67 @@
...
+- With *good sides together*, matching notches, pin the sleeve head along the armhole.
+- If necessary, ease in the sleeve head at the top, between the notches.
+- Sew, finish and press the seam.
+- Repeat for other sleeve.
 
-Sew, finish and press the seam.
+### Step 5: Close side and sleeve seam
 
-## Close side and sleeve seam
+- With *good sides together* pin the front to the back with along the sleeve and side seams, making sure the armhole seams line up.
```
Comment (nicholasdower) [inline markdown/org/docs/patterns/diana/instructions/en.md:50]: I think the "with along" bit is a typo. Tried to fix but please check whether it makes sense.
```suggestion
- With *good sides together* pin the front to the back with the sleeve and side seams matching, making sure the armhole seams line up across the front and back.
```

File: markdown/org/docs/patterns/diana/options/acrossbackfactor/en.md [hunk 3]
```diff
@@ -1,7 +1,6 @@
-![The across back option on Brian](./acrossbackfactor.svg)
```
[the remark this answers is not in the document]
Comment (bobgeorgethe3rd) [reply markdown/org/docs/patterns/diana/options/acrossbackfactor/en.md:1]: They are of Brian not of Diana. Originally patterns would share options pages if they had the same option. When the markdown was merged into the monorepo it allowed for svg images of what the options do to be generated directly from the patterns so it required the pattern option pages to be split/duplicated hence the holdover of some of Brians. I actually have been meaning to talk to you about this as there is an issue where some of these pages still display the incorrect options due to the split not being fully realised. #1342 has the list and I have been meaning to make a separate issue for it.

File: markdown/org/docs/patterns/diana/instructions/en.md (hunk 2 above)
Comment (bobgeorgethe3rd) [reply markdown/org/docs/patterns/diana/instructions/en.md:50]: mm let me fix this

File: markdown/org/docs/patterns/diana/options/acrossbackfactor/en.md (hunk 3 above)
[the remark this answers is not in the document]
Comment (nicholasdower) [reply markdown/org/docs/patterns/diana/options/acrossbackfactor/en.md:1]: Oh I see! Diana doesn't look at all like the garment in these photos. Thank you very much for clarifying.

**Review by nicholasdower — APPROVED** [review]
Great update! 🧷

File: markdown/org/docs/patterns/diana/instructions/en.md (hunk 1 above)
Comment (bobgeorgethe3rd) [reply markdown/org/docs/patterns/diana/instructions/en.md:23]: Yea I can see that one sec.

Comment (nicholasdower) [reply markdown/org/docs/patterns/diana/instructions/en.md]: 🙏

**Pull request merged**